<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>153</width>
      <height>603</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCustomLabel" name="LB_cabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Кабели на складе</string>
     </property>
    </widget>
    <widget class="QCustomLabel" name="LB_Have">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>66</y>
       <width>113</width>
       <height>17</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Выдача кабеля</string>
     </property>
    </widget>
    <widget class="QCustomLabel" name="LB_back">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>92</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Возврат кабеля</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="TAB_MainFlow">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>-28</y>
      <width>645</width>
      <height>629</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QTableWidget" name="TW_cabellist">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>645</width>
        <height>519</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Номер кабеля</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="PHB_addnewcabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить новый кабель</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>475</x>
        <y>15</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Кабели на складе</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="TW_havelist">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>645</width>
        <height>521</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Номер кабеля</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ФИО берущего</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Кол-во взятое</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="PHB_addnewhave">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить новую запись</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>475</x>
        <y>15</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выдача кабелей</string>
      </property>
     </widget>
     <zorder>PHB_addnewhave</zorder>
     <zorder>label_2</zorder>
     <zorder>TW_havelist</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Страница</string>
     </attribute>
     <widget class="QTableWidget" name="TW_backlist">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>645</width>
        <height>521</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>true</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Дата</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Номер кабеля</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ФИО берущего</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Кол-во к возрату</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="PHB_addnewback">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>200</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить новую запись</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>475</x>
        <y>15</y>
        <width>160</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Condensed</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Возврат кабелей</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomLabel</class>
   <extends>QLabel</extends>
   <header>qcustomlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
